src/DynamoCLI/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dynamo.Applications.StartupUtils;
+using System.Diagnostics;
+using Dynamo.Models;
+using Dynamo.Services;
+using System.Text.RegularExpressions;
+
+namespace DynamoCLI
+{
+    internal class Program
+    {
+
+        protected static CommandLineArguments commandstringToArgs(string commandstring)
+        {
+            var argarray = commandstringToStringArray(commandstring);
+            var args = CommandLineArguments.FromArguments(argarray);
+            return args;
+        }
+
+        protected static string[] commandstringToStringArray(string commandstring)
+        {
+            //convert string to commandlineargs, 
+            var m = Regex.Matches(commandstring, "([^\"]\\S*|\".+?\")\\s*");
+            var list = m.Cast<Match>().Select(match => match.Value).ToList();
+
+            //strip trailing whitespace
+            var argarray = list.Select(x => x.Trim()).ToArray();
+            return argarray.ToArray();
+        }
+
+
+
+        [STAThread]
+        static internal void Main(string[] args)
+        {
+
+            try
+            {
+                var cmdLineArgs = CommandLineArguments.FromArguments(args);
```

Comment:
This doesn't really roll off the tongue. Rename `CommandLineArguments.FromArguments` to `CommandLineArguments.Parse`. This is also more similar to the `Parse` method available on c# types.